docs/hub/_toctree.yml

```diff
@@ -137,6 +137,8 @@
         title: Integrate a library with the Hub
       - local: models-tasks
         title: Tasks
+      - local: dduf
+        title: DDUF
```

Comment:
maybe after GGUF

Reply:
done in https://github.com/huggingface/hub-docs/pull/1526/commits/3c5b4b1343ae7d62a9d08d19675affc7bcc56e2d